### DISCUSSION TITLE
what should be redirect uri for google auth

### DISCUSSION THREAD

**Discussion by abhinavgautam07** [provectus/kafka-ui#3175] [Q&A]
<img width="626" alt="image" src="https://user-images.githubusercontent.com/49569646/209894622-126df4ed-3c9e-4847-abbf-280fc97439be.png">
<img width="556" alt="image" src="https://user-images.githubusercontent.com/49569646/209894651-8afd46d0-7b09-466c-ac66-58e226b0215e.png">

**Comment by Haarolean** [comment]
Duplicate of #3183?
